build.cmd

```diff
@@ -314,6 +316,7 @@ REM ============================================================================
 echo %__MsgPrefix%Using environment: "%__VSToolsRoot%\VsDevCmd.bat"
```

Comment:
orthogonal to your changes but I don't think this code is even needed, we should probably look closer at all the places calling vsdevcmd.bat (both in here and in run.cmd) and make them more consistent.

Reply:
What is your suggestion now? Should we check VsDevCmd.bat invocations in this PR or should I open new issue to track VsDevCmd.bat usage review and comment on this changes to remove them once they will not be required?

Reply:
Unless you want to don't do it as part of this PR, but having an issue would be good if you don't want to tackle it. It is just something I noticed while reviewing your change.

Reply:
Will do it and I have created issue #14540 to track it's progress - just assign it to me.